lib/mongo/auth/user.rb

```diff
@@ -131,6 +131,7 @@ def hashed_password
       # @option options [ String ] :password The user's password.
       # @option options [ Symbol ] :auth_mech The authorization mechanism.
       # @option options [ Array<String>, Array<Hash> ] roles The user roles.
+      #@option options [ String ] :client_key The user's client key cached from a previous authentication on the same connection.
```

Comment:
minor nitpick: spacing after the ``#`` and usually we don't let lines get as long as this one.